### PR TITLE
Update CSS minmax() page macros and links

### PR DESCRIPTION
**Pull request by AnilSeervi**
- `http` to `https`
- use macros

### CODE REVIEW

File: files/en-us/web/css/minmax()/index.html [hunk 1]
```diff
@@ -79,10 +79,10 @@ <h3 id="CSS_properties">CSS properties</h3>
...
- <li><a href="/en-US/docs/Web/CSS/grid-template-rows">grid-template-rows</a></li>
- <li><a href="/en-US/docs/Web/CSS/grid-auto-columns">grid-auto-columns</a></li>
- <li><a href="/en-US/docs/Web/CSS/grid-auto-rows">grid-auto-rows</a></li>
+ <li>{{CSSxRef("grid-template-columns")}}</li>
+ <li>{{CSSxRef("grid-template-rows")}}</li>
+ <li>{{CSSxRef("grid-auto-columns")}}</li>
+ <li>{{CSSxRef("grid-auto-rows")}}</li>
 </ul>
 
 <h2 id="Examples">Examples</h2>
```
Comment (AnilSeervi) [inline files/en-us/web/css/minmax()/index.html:88]: Does the `hightlight[3]` class have any importance now; at line 92
Also what about the `no-line-numbers` class at line 24

Comment (teoli2003) [reply files/en-us/web/css/minmax()/index.html:88]: No `highlight[3]` was for a feature long gone. And `no-line-numbers` was never supported.

We do remove both of them.

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Just the two classes you asked about to remove and we are good to go.

File: files/en-us/web/css/minmax()/index.html (hunk 1 above)
Comment (AnilSeervi) [reply files/en-us/web/css/minmax()/index.html:88]: Sure, removed them.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**